src/schemas/22-8794/schema.js

```diff
@@ -0,0 +1,146 @@
+import _ from 'lodash';
+import definitions from '../../common/definitions';
+
+const origDefinitions = _.cloneDeep(definitions);
+const pickedDefinitions = _.pick(origDefinitions, ['date', 'fullNameNoSuffix', 'usaPhone', 'yesNoSchema', 'email']);
+
+const textAndNumbersRegex = '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*]+$';
+const schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'Designation Of Certifying Official(S) (22-8794)',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.merge(origDefinitions, pickedDefinitions),
+  required: [
+    'designatingOfficial',
+    'institutionDetails',
+    'primaryCertifyingOfficial',
+    'statementOfTruthSignature',
+    'dateSigned',
+  ],
+  properties: {
+    designatingOfficial: {
+      type: 'object',
+      required: ['fullName', 'title', 'emailAddress'],
+      properties: {
+        fullName: { $ref: '#/definitions/fullNameNoSuffix' },
+        title: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 50,
+          pattern: '^(?!\\s)(?!.*?\\s{2,})[a-zA-Z0-9]+$',
+        },
```

Comment:
[a-zA-Z0-9]+: One or more alphanumeric characters only (no spaces, punctuation, or special characters). Should the title not have spaces? "Foo Title" and 'Vice President' would fail.